views/index.js

```diff
@@ -32,7 +32,14 @@ exports.saveRequest = function(req, res) {
     }).then( function(numRequests) {
         requestData = utils.getRequestData(req, numRequests, region_code);
         requestData = utils.createSerial(numRequests, requestData, region_code);
-        requestData.is_sms = 'web';
+        
+        // Check for 311 origin:
+        if (req.url == '/311') {
```

Comment:
The general-vs-specific thing here too -- again, I don't think we need to do anything about it right now, since we told Jim the route would be "/311" and we can future-proof it later (if more cities start doing this, then "/311" will just redirect to "/chi-311" or something, and no one will get the generic "/311" as their advertised URL anymore).